src/fairseq2/models/s2t_transformer/builder.py

```diff
@@ -90,6 +90,9 @@ class S2TTransformerConfig:
     depthwise_conv_kernel_size: int
     """The kernel size of depthwise convolutions in Conformer blocks."""
 
+    conv_norm_type: Literal["batch_norm", "layer_norm"]
```

Comment:
I don't think we should expose this as a configuration parameter for S2T Transformer. That model architecture has never used Conformer with LayerNorm and will like never use it.